generator/routing_generator.hpp

```diff
@@ -14,5 +14,8 @@ void BuildRoutingIndex(string const & baseDir, string const & countryName, strin
 /// @param[in]  baseDir      Full path to .mwm files directory.
 /// @param[in]  countryName   Country name same with .mwm and .border file name.
 /// @param[in]  osrmFile  Full path to .osrm file (all prepared osrm files should be there).
-void BuildCrossRoutingIndex(string const & baseDir, string const & countryName, string const & osrmFile);
+/// @param[in]  warningsFile File path to write border intersection warnings. Checks will not
```

Comment:
File path for border intersection warnings. If corresponding file exists, old contents will be overwritten, otherwise a new file will be created.